crates/bevy_pbr/src/light_probe/environment_filter.wgsl

```diff
@@ -0,0 +1,427 @@
+#import bevy_render::maths::{PI, PI_2};
+#import bevy_pbr::lighting::perceptualRoughnessToRoughness;
+
+struct PrefilterConstants {
+    mip_level: f32,
+    sample_count: u32,
+    roughness: f32,
+    blue_noise_size: vec2f,
+}
+
+@group(0) @binding(0) var input_texture: texture_2d_array<f32>;
+@group(0) @binding(1) var input_sampler: sampler;
+@group(0) @binding(2) var output_texture: texture_storage_2d_array<rgba16float, write>;
+@group(0) @binding(3) var<uniform> constants: PrefilterConstants;
+@group(0) @binding(4) var blue_noise_texture: texture_2d<f32>;
+
+// Tonemapping functions to reduce fireflies
+
+const white_point: f32 = 1.0;
```

Comment:
What's the white point calculation here? It's different from what I used in TAA, where was the math derived from?